pallets/ddc-staking/src/lib.rs

```diff
@@ -329,6 +329,11 @@ pub mod pallet {
 	#[pallet::getter(fn pricing)]
 	pub type Pricing<T: Config> = StorageValue<_, u128>;
 
+	/// A list of accounts allowed to become cluster managers.
+	#[pallet::storage]
+	#[pallet::getter(fn cluster_managers)]
+	pub type ClusterManagers<T: Config> = StorageValue<_, Vec<T::AccountId>, ValueQuery>;
```

Comment:
General question, how this data structure is integrated into current functionality? what is the goal of a cluster manager?

Reply:
It is a list of accounts that governance allows to become a cluster manager. The list is required for authorization in off-chain systems. Further development will introduce depositing to get into the list.